VENDORING.md

```diff
@@ -0,0 +1,235 @@
+build failure that could result from a package change on the upstream.
+
+### Update a vendored package
+
+This case describes a situation where the developer adds some code relying on
+another version of a package already vendored. This is a very simple case since
+the package is already part of the vendoring. The developer will have to modify
+`Gopkg.toml` by updating the vendoring version:
+
+```diff
+--- Gopkg.toml.orig     2018-01-23 09:04:54.160760426 -0800
++++ Gopkg.toml.new      2018-01-23 09:05:09.164094911 -0800
+@@ -1,3 +1,3 @@
+ [[constraint]]
+   name = "github.com/foo/bar"
+-  revision = "f5742cb6"
++  revision = "a4be45c7"
+```
+
+And let `dep` update the vendoring by relying on this new revision:
+```bash
+$ dep ensure
+$ dep prune
+```
+
+### Remove a vendored package
+
+This case describes a situation where the developer modifies the existing
+code, no longer relying on a vendored package. Running the usual commands
+will remove the vendored package from `vendor` directory and from
+`Gopkg.lock`:
+```bash
+$ dep ensure
+$ dep prune
+```
+
+If a constraint has been set in `Gopkg.toml`, this is the responsibility of
+developer to remove it manually from this file.
+
+### Test a pending pull request
+
+This case describes a situation where the developer needs to perform some
+testing on his code, depending on a pending pull request from a vendored
+package. The easiest and safest way to do that is to rely on a feature proposed
+by `dep` called `override`. This allows the developer to leave the constraints
+unmodified but to rely on a different version of the vendored package for
+testing purpose.
+
+#### From the main repository
+
+The pending pull request can be submitted to the main repository, through a
+development branch `work-dev`, in which case the developer will have to modify
+`Gopkg.toml` as follow:
+
+```diff
+--- Gopkg.toml.orig     2018-01-23 09:04:54.160760426 -0800
++++ Gopkg.toml.new      2018-01-23 09:06:49.735633780 -0800
+@@ -1,3 +1,7 @@
+ [[constraint]]
+   name = "github.com/foo/bar"
+   revision = "f5742cb6"
++
++[[override]]
++  name = "github.com/foo/bar"
++  branch = "work-dev"
+```
+
+#### From a forked repository
+
+The pending pull request can be submitted to a forked repository, through
+a development branch called `work-dev`, in which case the developer will have
+to modify `Gopkg.toml` as follow:
+
+```diff
+--- Gopkg.toml.orig     2018-01-23 09:04:54.160760426 -0800
++++ Gopkg.toml.new      2018-01-23 09:09:58.475261707 -0800
+@@ -1,3 +1,8 @@
+ [[constraint]]
+   name = "github.com/foo/bar"
+   revision = "f5742cb6"
++
++[[override]]
++  name = "github.com/foo/bar"
++  branch = "work-dev"
++  source = "github.com/myfork/bar"
+```
+
+It is more common to see this case happening. Usually, developers don't have
+permissions to push directly to the main repository.
+
+## Miscellaneaous
+
+Take a look [here](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#re-vendor-prs)
```

Comment:
Lines 234-235 suggested rewrite: For additional information about vendoring, see "[Re-vendor PRs](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#re-vendor-prs)".

Reply:
done